package.json

```diff
@@ -50,8 +50,8 @@
     ]
   },
   "peerDependencies": {
-    "react": ">=15.0.0",
-    "react-dom": ">=15.0.0"
+    "react": ">=16.6.0",
```

Comment:
We can kill `react-lifecycles-compat`, now please!
https://bundlephobia.com/result?p=react-lifecycles-compat@3.0.4